### PR TITLE
Update CHT translation

### PR DESCRIPTION
**Pull request by protonchang**
* Add new translations

### CODE REVIEW

**Comment by sshane** [comment]
@eFiniLan does this look good to you?

**Comment by protonchang** [comment]
Found a small typo in the new entry
Fixed in 1657ec9

**Comment by eFiniLan** [comment]
> @eFiniLan does this look good to you?

1 minor change as per comment, otherwise looking good :)

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 1]
```diff
@@ -592,16 +592,16 @@ location set</source>
...
     <message>
         <source>Press confirm to erase all content and settings. Press cancel to resume boot.</source>
-        <translation type="unfinished"></translation>
+        <translation>按下確認以刪除所有內容及設定 按下取消來繼續開機</translation>
     </message>
     <message>
         <source>Resetting device...
 This may take up to a minute.</source>
-        <translation type="unfinished"></translation>
+        <translation>裝置重置中 此過程可能需要幾分鐘</translation>
```
[the remark this answers is not in the document]
Comment (protonchang) [reply selfdrive/ui/translations/main_zh-CHT.ts:604]: [https://github.com/commaai/openpilot/blob/master/selfdrive/ui/translations/main_zh-CHT.ts#L203](https://github.com/commaai/openpilot/blob/master/selfdrive/ui/translations/main_zh-CHT.ts#L203)

**Comment by eFiniLan** [comment]
Yeah, that will be great 👍

**Comment by protonchang** [comment]
All string updated

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 2]
```diff
@@ -1028,11 +1028,11 @@ This may take up to a minute.</source>
     </message>
     <message>
         <source>openpilot longitudinal control may come in a future update.</source>
-        <translation type="unfinished"></translation>
+        <translation>未來可能會推出Openpilot縱向控制</translation>
```
Comment (eFiniLan) [inline selfdrive/ui/translations/main_zh-CHT.ts:1031]: "openpilot" has to be all in lower case I think?

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 3]
```diff
@@ -1028,11 +1028,11 @@ This may take up to a minute.</source>
...
     </message>
     <message>
         <source>An experimental version of openpilot longitudinal control can be tested, along with Experimental mode, on non-release branches.</source>
-        <translation type="unfinished"></translation>
+        <translation>在非發行分支中 可找到包含實驗模式的Openpilot縱向控制測試版本</translation>
```
Comment (eFiniLan) [inline selfdrive/ui/translations/main_zh-CHT.ts:1035]: same as above, use "openpilot" instead :)

**Comment by eFiniLan** [comment]
thanks @protonchang for the changes, @sshane it's good to go 👍

**Pull request merged**